.github/PULL_REQUEST_TEMPLATE/python.md

```diff
@@ -0,0 +1,169 @@
+**Changes:**
+
+* IoT3 SDK:
+    * core: ...
+    * mobility: ...
+* applications:
+    * iqm: ...
+    * ...
+
+Closes: #
+
+---
+**How to test:**
+
+_**Note:** in the following, lines starting with `$` are to be executed on your machine, as a non-root user; lines starting with `(docker)$` are to be executed in the Docker container; lines starting with `(docker)🐍 $` are to be executed in the Docker container, in the python venv._
```

Comment:
'(python) instead of (docker)?

Reply:
It's `(docker)🐍 $` to indicate that it is a command to run in the docker container (`(docker)`) in the python venv (`🐍`) as a user (`$`).